CONTRIBUTING.md

```diff
@@ -0,0 +1,619 @@
+
+- Describing *interactions* with the user interface for a procedure or task, see: [Microsoft Style Guide: Describing interactions with UI](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui).
+
+- Describing *interactions* with the user interface for a procedure or task with multiple input methods; for example, procedures that can be performed with a keyboard or a mouse; see: [Microsoft Style Guide: Describing alternative input methods - Multiple input methods and branching within procedures](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#multiple-input-methods-and-branching-within-procedures).
+
+- Referring to user interface elements in instructions or procedures, see: [Microsoft Style Guide: Formatting text in instructions](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions).
+
+- Referring to the mouse and mouse-based interactions, such as "click" and "right mouse button"; see: [Microsoft Style Guide: Mouse and mouse interaction terms](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/mouse-mouse-interaction-terms).
+
+- Referring to keyboard keys and keyboard shortcuts, see: [Microsoft Style Guide: Keys and keyboard shortcuts](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts).
+
+- Referring to touch interactions, such as "tap" and "swipe"; see: [Microsoft Style Guide: Describing alternative input methods - Pen-computing and touch procedures](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#pen-computing-and-touch-procedures).
+
+## Adding admonitions or notes
+
+Avoid adding admonitions (also known as: *notices*, *notes*, *warnings*, or *callouts*) where possible. Over-use of admonitions may lead to what is sometimes referred to as "Admonition Fatigue". For an article on reason for limiting the use of admonitions, see: [Harvard Business Review: Consumer Warning Labels Aren’t Working](https://hbr.org/2016/11/consumer-warning-labels-arent-working).
+
+Admontions typically start with: *Tip*, *Note*, *Important*, *Caution*, or *Warning*.
+
+For example:
+
+This is a note; used for additional, indirectly related information.
+
+This is a note that is important for users to read.
+
+Tiny documentation uses four levels of admonition:
+
+1.  NOTE: Use for additional, indirectly related information. Do not use where it is possible to reword or rewrite the content to incorporate the information.
+
+2.  IMPORTANT: Use when ignoring the notice *may* or *will* lead to unexpected behavior.
+
+3.  CAUTION: Use when ignoring the notice *may* lead to one or more of the following:
+
+    - A significant increase in the risk of a security breach.
+
+    - Create a security vulnerability.
+
+    - Cause information loss.
```

Comment:
```suggestion
    - Information loss.
```

For each item in the list that follows _one or more of the following_ to, in fact, **be** _one or more of the following_, said items must be things: nouns or noun phrases. _Cause information loss_ is an action; a verb phrase.